c7n/resources/route53.py

```diff
@@ -485,3 +486,42 @@ def process(self, resources, event=None):
             elif not logging and not state:
                 results.append(r)
         return results
+
+
+@HostedZone.filter_registry.register('enhance-hostedzone-attributes')
```

Comment:
@sgungi-intuit is this supposed to be in this PR? the description of the PR only mentions rds, but here I see code for route53. If this is meant to be here, then please update PR description and also add unit test. If not, then remove it.